core/src/test/java/org/apache/carbondata/core/load/LoadMetadataDetailsUnitTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.carbondata.core.load;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class LoadMetadataDetailsUnitTest {
+
+  private LoadMetadataDetails loadMetadataDetails;
+
+  @Before public void setup() {
+    loadMetadataDetails = new LoadMetadataDetails();
+  }
+
+  /**
+   * This method will test Hashcode which will return 31 if we don't set loadName.
+   *
+   * @throws Exception
+   */
+
+  @Test public void testHashCodeLoadNameNull() throws Exception {
+    Integer data = loadMetadataDetails.hashCode();
+    assert (data == 31);
```

Comment:
better to use assertEquals, so that log will give the actual value in case of failure